### DISCUSSION TITLE
v2.4.0

### DISCUSSION THREAD

**Discussion by InfinityLoop1308** [InfinityLoop1308/PipePipe#66] [Announcements]
feat: support filtering play histories
feat: support searching playlists(NicoNico)
fix: add consent header for every YouTube request to possibly fix #14
fix: under specific situations app crash on start forever
fix: search icon is not visible in light mode
behaviour: improve url handling in descriptions and comments to fix #64
UI: show single P name at first for multi-P videos(BiliBili)

<hr /><em>This discussion was created from the release <a href='https://github.com/InfinityLoop1308/AnimePipe/releases/tag/'>v2.4.0</a>.</em>

**Comment by straw44** [comment]
https://github.com/InfinityLoop1308/AnimePipe/issues/64#issuecomment-1421011394
